sdks/python/apache_beam/io/avroio.py

```diff
@@ -510,8 +301,6 @@ def __init__(
         is '-SSSSS-of-NNNNN' if None is passed as the shard_name_template.
       mime_type: The MIME type to use for the produced files, if the filesystem
         supports specifying MIME types.
-      use_fastavro (bool); when set, use the `fastavro` library for IO, which
```

Comment:
As a courtesy to users, let's keep this flag in user-facing public APIs, but make it have no effect.

Otherwise, when users update to a new SDK, their pipeline will start to fail at startup and will require changes. This is disruptive. Instead, let's say in docstrings (where applicable):
```use_fastavro: This flag is left for API backwards compatibility and no longer has an effect. Do not use. ```

Reply:
We can still remove the flag from internal code that is not subject to backwards-compatibility.

Reply:
Okay. So, it will be just passed as an input parameter so that the code won't break if user uses this flag but irrespective of true/false, we would use fastavro right?